device-types/Lenovo/ThinkSystem-SR635.yaml

```diff
@@ -0,0 +1,17 @@
+---
+manufacturer: Lenovo
+model: ThinkSystem SR635
+slug: lenovo-thinksystem-sr635
+part_number: 7Y98
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
+interfaces:
+  - name: XCC
+    type: 1000base-t
+    mgmt_only: true
+power-ports:
+  - name: PSU0
```

Comment:
As these are Modular PSUs they need to be modeled as `module-bays` and then add the psu's in the modules folder if they don't already exist.